### PR TITLE
PRs 5202/5202, 5212, 5215, 5223, 5224

### PR DESCRIPTION
**Pull request by peiying2**
*5203: Post launch of Dev Quantum
*5212: Firefox Quantum redesign
*5223: Firefox/features/index update
*5224: Firefox mobile

Activation under firefox/share.lang
*firefox/hub/home-quantum.lang

Tags
*firefox/products/developer-quantum.lang

Activation
*firefox/mobile.lang

### CODE REVIEW

**Comment by peiying2** [comment]
PR 5212 contain strings small leverageable from these files
-- firefox/new/quantum.lang
-- firefox/products/developer-quantum.lang
--firefox/sync.lang (this is obsolete, but not removed, so we can still leverage, right?)

Staging URL: I use the standard one. It may not be available by the time we release the strings for l10n.

Should the firefox/home.lang be obsolete?

**Comment by peiying2** [comment]
PR5203: I found a dup string in the file and removed en-US links released previously.

**Comment by peiying2** [comment]
PR5202 seems to be the same as PR5203 content-wise. I compared the changes and they are off by two lines, but they don't seem to be l10n related.

**Comment by flodolo** [comment]
> PR5202 seems to be the same as PR5203 content-wise. I compared the changes and they are off by two lines, but they don't seem to be l10n related.

Yes, PR5202 and PR5203 are expected to have the same content. Files changes look good.

File: en-US/firefox/products/developer-quantum.lang [hunk 1]
```diff
@@ -210,10 +295,6 @@ Welcome to the all-new Firefox Quantum: Developer Edition
 Firefox has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
 
 
-;Firefox Devtools
-Firefox Devtools
```
Comment (flodolo) [inline en-US/firefox/products/developer-quantum.lang]: Please add this back. Remember that the errors page shows you if there are duplicate strings, and this is not (check the case).

File: en-US/firefox/hub/home-quantum.lang [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+The new, fast browser for Mac, PC and Linux
+
+
+# HTML page description
+;Responsive engine, less memory usage and packed with features. Download for desktop now.
+Responsive engine, less memory usage and packed with features. Download for desktop now.
+
+
+;Firefox Quantum
+Firefox Quantum
```
Comment (flodolo) [inline en-US/firefox/hub/home-quantum.lang]: I don't think we want to expose this for localization.

File: en-US/firefox/hub/home-quantum.lang [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+;Smooth browsing
+Smooth browsing
+
+
+;Whether you’ve opened 10 or 1000, switching between tabs is faster than ever with Firefox’s new, responsive engine.
+Whether you’ve opened 10 or 1000, switching between tabs is faster than ever with Firefox’s new, responsive engine.
+
+
+;Firefox screenshot showing many open tabs and browser windows running simultaneously.
```
Comment (flodolo) [inline en-US/firefox/hub/home-quantum.lang]: Add a note that this is an alternative text for a screenshot

File: en-US/firefox/hub/home-quantum.lang [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+;“Faster and even more minimalist, Firefox Quantum makes Chrome look old.”
+“Faster and even more minimalist, Firefox Quantum makes Chrome look old.”
+
+
+# Link points to https://www.digitaltrends.com/web/firefox-quantum-hands-on-review/
+;(Digital Trends)
+(Digital Trends)
+
+
+;Screenshot of the redesigned Firefox new-tab page.
```
Comment (flodolo) [inline en-US/firefox/hub/home-quantum.lang]: Add a note that this is an alternative text for a screenshot

**Comment by flodolo** [comment]
> Should the firefox/home.lang be obsolete?

Yes



> Activation under firefox/share.lang

Remember to add it to your langchecker PR
https://github.com/mozilla-l10n/langchecker/blob/master/app/scripts/activate_firefox_shared

File: en-US/firefox/products/developer-quantum.lang [hunk 5]
```diff
@@ -210,8 +295,8 @@ Welcome to the all-new Firefox Quantum: Developer Edition
...
+;;Firefox DevTools
+Firefox DevTools
```
Comment (flodolo) [inline en-US/firefox/products/developer-quantum.lang]: Uhm, any error on the errors page? Because you removed "Devtools" and added back "DevTools"

**Comment by flodolo** [comment]
I think we're good. Later today I'll add the other languages and import strings where possible (make sure we mention that in the comms).

**Comment by flodolo** [comment]
And just noticed PR5224. I'll take a look and see if I can include it here.

**Comment by flodolo** [comment]
Added mobile, updated a few DEMO urls (note that URL demo are usually available in the first message of a PR). I didn't import anything, even German only had a couple of strings. They can use Pontoon's machinery.

File: en-US/firefox/products/developer-quantum.lang [hunk 6]
```diff
@@ -295,8 +295,8 @@ Welcome to the all-new Firefox Quantum: Developer Edition
 Firefox has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
 
 
-;;Firefox DevTools
-Firefox DevTools
+;;Firefox Devtools
```
Comment (flodolo) [inline en-US/firefox/products/developer-quantum.lang]: Argh, double ;

**Comment by flodolo** [comment]
This covers also 5223 (I moved the 5 strings from new/onboarding to new/quantum)

**Comment by peiying2** [comment]
Thanks for adding the mobile file, that's what I was waiting for.  Checked the other commits, look good.

On URL, I did notice it in the first comment of the PR. Some look very temporary with personal name on it. I don't know how long the setup will last or they will move to the "standard" location.  But in the future, I will use it.

**Pull request merged**